breeze/breeze.py

```diff
@@ -30,6 +30,8 @@
     CONTRIBUTIONS='/api/giving',
     FUNDS='/api/funds',
     PLEDGES='/api/pledges',
+    TAGS='/api/tags/list_tags/',
```

Comment:
This ENDPOINTS enum is just meant to be the higher-level URL. In this case, would you mind removing  line 34

```
TAG_FOLDERS='/api/tags/list_folders/',
``` 

and then updating line 33 to be:

```
TAGS='/api/tags
```